packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 - Added `Gui.reportProgress` that can be used to notify users of action progress in conjunction with the `Gui.withProgress` call. [#2167](https://github.com/zowe/vscode-extension-for-zowe/issues/2167)
 - Updated linter rules and addressed linter errors throughout the codebase. [#2184](https://github.com/zowe/vscode-extension-for-zowe/issues/2184)
+- Added checks to verify that `@zowe/cli` dependency exists, after installing the API and during building. [#2199](https://github.com/zowe/vscode-extension-for-zowe/issues/2199)
```

Comment:
Instead of "after installing the API and during building", should this say "before building" now that the postinstall script has been removed?